cli/flox/src/commands/init/node.rs

```diff
@@ -377,24 +547,101 @@ impl Node {
+
+                    Would you like Flox to apply this suggestion?
+                    You can always change the environment's manifest with 'flox edit'
+                "}
+        };
+
+        let dialog = Dialog {
+            message: &message,
+            help_message: Some(AUTO_SETUP_HINT),
+            typed: Select {
+                options: ["Yes", "No", "Show suggested modifications"].to_vec(),
+            },
+        };
+        let (mut choice, _) = dialog.raw_prompt()?;
+
+        if choice == 2 {
+            let message = formatdoc! {"
+
+                {}
+                Would you like Flox to apply these modifications?
+                You can always change the environment's manifest with 'flox edit'
+            ", format_customization(&self.get_init_customization())?};
+
+            let dialog = Dialog {
+                message: &message,
+                help_message: Some(AUTO_SETUP_HINT),
+                typed: Select {
+                    options: ["Yes", "No"].to_vec(),
+                },
+            };
+
+            (choice, _) = dialog.raw_prompt()?;
+        }
+        Ok(choice == 0)
+    }
+
+    /// Prompt whether to install npm or yarn when either is viable
+    fn prompt_for_package_manager(
+        &mut self,
+        yarn_install: YarnInstall,
+        node_install: NodeInstall,
+        flox: &Flox,
+    ) -> Result<bool> {
+        let yarn_version = yarn_install
+            .yarn
+            .version
+            .as_ref()
+            .map(|version| format!(" {version}"))
+            .unwrap_or("".to_string());
+        let node_version = match &node_install.node {
+            Some(found_node) => found_node.clone(),
+            None => Self::get_default_package("nodejs", flox)?,
+        }
+        .version
+        .as_ref()
+        .map(|version| format!(" {version}"))
+        .unwrap_or("".to_string());
+
         let message = formatdoc! {"
-            {nodejs_detected}
             Flox detected both a package-lock.json and a yarn.lock
 
             Flox can add the following to your environment:
-            * nodejs{}
+            * Either nodejs{node_version} with npm bundled, or yarn{yarn_version} with nodejs{node_version} bundled
             * Either an npm or yarn installation hook
 
             Would you like Flox to apply one of these modifications?
-            You can always change the environment's manifest with 'flox edit'", nodejs_version.map(|version| format!(" {version}")).unwrap_or("".to_string())};
+            You can always change the environment's manifest with 'flox edit'"};
         let options = [
-            "Yes - with npm hook",
-            "Yes - with yarn hook",
+            "Yes - with npm",
+            "Yes - with yarn",
             "No",
-            "Show modifications with npm hook",
-            "Show modifications with yarn hook",
+            "Show modifications with npm",
+            "Show modifications with yarn",
```

Comment:
@jennymahmoudi
```
! Flox detected both a package-lock.json and a yarn.lock

Flox can add the following to your environment:
* Either nodejs 20.11.1 with npm bundled, or yarn 1.22.19 with nodejs 20.11.1 bundled
* Either an npm or yarn installation hook

Would you like Flox to apply one of these modifications?
You can always change the environment's manifest with 'flox edit'
> Yes - with npm
  Yes - with yarn
  No
  Show modifications with npm
  Show modifications with yarn
```

Reply:
Wording is great here. Minor format suggestion to help see the two choices.

IIRC, all of this text is bolded with the way our interactive prompts work, is that right? If yes, (it's all bold), let's emphasize the `or` with caps: 
> **Either nodejs 20.11.1 with npm bundled, OR yarn 1.22.19 with nodejs 18.19.1 bundled**


If not, I would suggest using bold for "Either" and both bold and caps for "or" in the choices:

> **Either** nodejs 20.11.1 with npm bundled, **OR** yarn 1.22.19 with nodejs 18.19.1 bundled

Reply:
I'm thinking it might be good to change the second line to `Either an npm installation hook, OR a yarn installation hook` then?